bin/dev

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# GEM_HOME/$(echo $RUBY_IMAGE | sed -e s/:/-/g)
+
+require 'optparse'
+
+options = {}
+OptionParser.new do |opts|
+  opts.banner = 'Usage: bin/dev [options] [command]'
+
+  opts.on('-iIMAGE', '--image=IMAGE', 'Specify Docker image') do |val|
+    options[:image] = val
+  end
+end.parse!
+
+IMAGE = options.fetch(:image, 'ruby:latest')
+IMAGE_PATH_SAFE = IMAGE.gsub(':', '_')
+IMAGE_NAME = "apm-agent-ruby:#{IMAGE_PATH_SAFE}"
+VENDOR_PATH = "/vendor/#{IMAGE_PATH_SAFE}"
+
+def run(cmd)
+  system "IMAGE_NAME=#{IMAGE_NAME} #{cmd}"
+end
+
+run 'docker-compose build ' \
```

Comment:
Will the docker container might change the source code somehow? If so, then it might be a good practice to match both userids, the one from the host and the one in the container when running the docker-compose run

Reply:
this is important, if you do not change the user, the build would create files in the folder `vendor` and `.` as the root user

Reply:
It should not change anything. Unless the dev opens an editor inside and changes things.

Reply:
I remember switching to the host's user in the old setup at some point too. What's the minimal approach to doing so?